client/client_test.go

```diff
@@ -1559,20 +1822,46 @@ func TestPublishSnapshotLocalKeysCreatedFirst(t *testing.T) {
 	assert.False(t, requestMade)
 }
 
+func createKey(t *testing.T, repo *NotaryRepository, role string, x509 bool) data.PublicKey {
+	key, err := repo.CryptoService.Create(role, data.ECDSAKey)
+	assert.NoError(t, err, "error creating key")
+
+	if x509 {
+		start := time.Now()
```

Comment:
When generating TLS certs, we should add a start time some time in the past. It makes it slightly safer (ensures that clock drifts don't make the cert not be valid).